tasks/ca/statcan/data.py

```diff
@@ -53,6 +53,22 @@ def download(self):
 
 
 class SplitAndTransposeData(BaseParams, Task):
+    INDIAN_RESERVES_SUFFIX = '-DQ'
+    DIVISION_COLUMN_NAMES = {
```

Comment:
Pitching in:
1. and 2. 
I am torn here. In one hand, I like Antonio's point that the parser should just parse a file, and receive a configuration. On the other hand, it is true that the parser already has a lot of configuration, albeit, hardcoded, and I don't like passing two separate parameters as two variables (specially since they are very closely related).

3. I am not a fan of overenginering, simplest solution is best and so on. But changing the string by a regex is quite simple and gives a lot of flexibility. I like it.
4. Agreed. It would be nice to have a method do the check if the row should be skipped in order to keep the same level of abstraction in the loop.